packages/test-utils/src/mount.js

```diff
@@ -13,22 +13,53 @@ import config from './config'
 import warnIfNoWindow from './warn-if-no-window'
 import createWrapper from './create-wrapper'
 import createLocalVue from './create-local-vue'
+import { semVerGreaterThan, warn } from 'shared/util'
+import { COMPAT_SYNC_MODE } from 'shared/consts'
+
 Vue.config.productionTip = false
 Vue.config.devtools = false
 
+function getSyncOption (syncOption) {
+  if (syncOption === false) {
+    Vue.config.async = true
+    return false
+  }
+  if (!semVerGreaterThan(Vue.version, '2.5.17')) {
+    warn(
+      `Vue Test Utils runs in sync mode by default. Due to bugs, sync mode ` +
+      `requires Vue > 2.5.18. In Vue Test Utils 1.0 sync mode will only be ` +
+      `supported with Vue 2.5.18 running in development mode. If you are ` +
+      `unable to upgrade, you should rewrite your tests to run synchronously` +
```

Comment:
I'm a little confused here: should this read

```suggestion
      `unable to upgrade, you should rewrite your tests to run asynchronously` +
```

?

Reply:
Yes, thanks Winnie 👍